bigquery_magics/__init__.py

```diff
@@ -29,11 +32,21 @@ def load_ipython_extension(ipython):
     )
     ipython.register_magic_function(_cell_magic, magic_kind="cell", magic_name="bqsql")
 
+    global is_registered
+    is_registered = True
+
+
+def unload_ipython_extension(ipython):
```

Comment:
Today, I learned!

https://ipython.readthedocs.io/en/stable/config/extensions/#writing-extensions

Might come in handy for undoing the bigframes-specific changes we're planning in https://github.com/googleapis/python-bigquery-dataframes/pull/1231